integration/integration_test.go

```diff
@@ -457,6 +509,10 @@ func TestOpensnoop(t *testing.T) {
 }
 
 func TestProcessCollector(t *testing.T) {
+	if *k8sDistro == K8sDistroARO {
+		t.Skip("Skip running process-collector gadget on ARO: iterators are not supported on kernel 4.18.0-305.19.1.el8_4.x86_64")
```

Comment:
I wonder if this should be detected with the kernel version, so that the test would be automatically enabled when the ARO kernel supports it.

```
kubectl get node -o jsonpath='{.items[0].status.nodeInfo.kernelVersion}'
```